### PR TITLE
Update to latlong2

### PR DESCRIPTION
**Pull request by Sipioteo**
This is just a reference changes to make this compatible with the last Flutter_Map update.
I had to remove the "popup" because it wasn't compatible.

### CODE REVIEW

File: example/ios/Flutter/flutter_export_environment.sh [hunk 1]
```diff
@@ -1,10 +1,13 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=/usr/share/flutter"
-export "FLUTTER_APPLICATION_PATH=/run/media/lorenzo/Volume/lorenzo/flutter_map_marker_cluster/example"
-export "FLUTTER_TARGET=lib/main.dart"
+export "FLUTTER_ROOT=E:\Programmi\Flutter"
```
Comment (klirix) [inline example/ios/Flutter/flutter_export_environment.sh:3]: is this needed in this pr?

File: lib/src/marker_cluster_layer_options.dart [hunk 2]
```diff
@@ -38,18 +37,6 @@ class AnimationsOptions {
   });
 }
 
-class PopupOptions {
```
Comment (klirix) [inline lib/src/marker_cluster_layer_options.dart:41]: This seems to be out of scope of this PR, maybe move it out into a separate one.

File: lib/src/marker_cluster_layer.dart [hunk 3]
```diff
@@ -543,17 +530,6 @@ class _MarkerClusterLayerState extends State<MarkerClusterLayer>
       layers.addAll(_buildLayer(layer));
     });
 
-    final PopupOptions popupOptions = widget.options.popupOptions;
```
Comment (klirix) [inline lib/src/marker_cluster_layer.dart:546]: Popups seem to be needed by cluster markers themselves

**Review by klirix — CHANGES_REQUESTED** [review]
There are some changes that might be out of scope.

Good job on update though :D

**Comment by TheOneWithTheBraid** [comment]
It seems like this PR is obsolete with #69.

**Pull request closed without merging**